### PR TITLE
Deploy i686 MSYS packages too, not only x86_64 ones

### PR DESCRIPTION
**Pull request by dscho**
[I noticed this morning](https://github.com/git-for-windows/MSYS2-packages/pull/70#issuecomment-1357489945) that `git-sdk-32` unexpectedly failed to update MinTTY to v3.6.3, and figured out that it was simply an oversight on my part in forgetting to implement the `/deploy` side of MSYS packages' builds, and therefore only the x86_64 package was built (and that one quite by happy mistake).

This PR introduces support for `/deploy`ing MSYS packages (the `git-for-windows-automation` workflow was [already prepared for it in advance](https://github.com/git-for-windows/git-for-windows-automation/commit/349e4e17451b3a8be428e8f81a62f57db7620efa).

### CODE REVIEW

**Comment by dscho** [comment]
Aaaargh! I forgot to merge this before deploying `curl`!

@rimrul could you give this a quick look-over while I manually deploy the i686 variant of `curl`?

**Comment by rimrul** [comment]
I've already looked at commit 1&3, but wanted to follow your advice about `-w` on number 2, but forgot to do that while at my PC. I guess I'll deal with the GitHub UI on my phone.

**Comment by dscho** [comment]
> wanted to follow your advice about `-w` on number 2

@rimrul here you go:

image

and with colored words:

image

**Comment by dscho** [comment]
> and with colored words:

This was done using the following alias in my `~/.gitconfig`:

```ini
[alias]
        cwshow = show --color-words=\"[A-Za-z0-9_]+|.\"
```

File: GitForWindowsHelper/slash-commands.js [hunk 1]
```diff
@@ -141,21 +141,33 @@ module.exports = async (context, req) => {
...
+                await appendToIssueComment(
+                    context,
+                    await getToken(),
+                    owner,
+                    repo,
+                    commentId,
+                    text
+                )
+            const answer = await triggerBuild()
+            const answer2 = appendToComment(`The workflow run [was started](${answer.html_url})`)
```
Comment (rimrul) [inline GitForWindowsHelper/slash-commands.js]: Don't we need await here?

Comment (dscho) [reply GitForWindowsHelper/slash-commands.js]: Absolutely!

Comment (dscho) [reply GitForWindowsHelper/slash-commands.js]: Fixed, in https://github.com/git-for-windows/gfw-helper-github-app/pull/14/commits/98372b343dd4ec9e5c24501fc83dd380629d71cb:

image

**Review by rimrul — APPROVED** [review]

**Pull request merged**